pyart/map/gates_to_grid.py

```diff
@@ -30,7 +30,6 @@ def map_gates_to_grid(
     gatefilters=False,
     map_roi=True,
     weighting_function="Barnes2",
-    toa=17000.0,
```

Comment:
Why remove the top of atmosphere constrant?

Reply:
That is the other contribution I wanted to separate. The toa variable was specified in this method but was not used/called even once, so I suspect that this is a residual from earlier releases, though I could be wrong.

Reply:
@mgrover1, of course. That is a good idea.
I'm attaching an example notebook demonstrating the solution for this type of case (gridding of a single PPI sweep).
[zdist_exmaple.zip](https://github.com/ARM-DOE/pyart/files/14000249/zdist_exmaple.zip)

Reply:
@isilber - can you add this as an example `.py` script? (see this as an example - https://github.com/ARM-DOE/pyart/blob/main/examples/mapping/plot_map_one_radar_to_grid.py), this will ensure it appears in the example gallery.

You can put it in the `/examples/plotting` directory, just make sure to call it something that starts with `plot_...` as that will ensure matplotlib sphinx builds it.

Reply:
@mgrover1 Where should the example file be placed?